app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file
+ *
+ * Success screen for the invite users modal.
+ */
+import * as React from 'react'
+
+import * as reactRouter from 'react-router'
+
+import ArrowRightIcon from 'enso-assets/arrow_right.svg'
+
+import * as appUtils from '#/appUtils'
+
+import * as textProvider from '#/providers/TextProvider'
+
+import * as ariaComponents from '#/components/AriaComponents'
+import * as result from '#/components/Result'
+
+/**
+ * Props for the InviteUsersSuccess component.
+ */
+export interface InviteUsersSuccessProps {
+  readonly onClose?: () => void
+  readonly emails: string[]
+  readonly invitationLink: string
+}
+
+/**
+ * The number of emails to display in the success message.
+ * If the number of emails is greater than this number, the message will be
+ * x emails have been invited instead of listing all the emails.
+ */
+const MANY_EMAILS = 4
+
+/**
+ * Success screen for the invite users modal.
+ */
+export function InviteUsersSuccess(props: InviteUsersSuccessProps) {
+  const { onClose, emails, invitationLink } = props
+  const { getText, locale } = textProvider.useText()
+  const membersSearchParams = 'cloud-ide_page="settings"&cloud-ide_SettingsTab="members"'
```

Comment:
Yeah, it's heavily fragile, and atm I see 2 solutions: either refactor settings to routes or use `@tanstack/router` since it provides typesafety for URLs

Reply:
i left a suggestion somewhere below suggesting to use an explicit `boolean` prop instead. might not be *ideal*, but at the very least it should be less fragile right?